src/Slices/ServiceInstanceDetails/UI/Tabs/ResourcesTabContent.tsx

```diff
@@ -43,14 +42,21 @@ export const ResourcesTabContent: React.FC = () => {
   }
 
   if (isError) {
-    return (
-      <TabContent role="tabpanel" id={"Resources-content"}>
-        <ErrorView
-          message={error.message}
-          ariaLabel="Error_view-Resources-content"
-        />
-      </TabContent>
-    );
+    // If the error is because of the version, we don't want to show the error view, and fall back to the loading view as the process of updating version is still ongoing
+    if (
+      !error.message.includes(
+        "Request conflicts with the current state of the resource: The given current version",
```

Comment:
Is this the only way? Doesn't seem like a sustainable solution.

Reply:
Added status code to error handling, and updated all queries to have that, to keep them aligned